### PR TITLE
OCPBUGS#34836: Update the Hybrid Helm project layout table

### PR DESCRIPTION
**Pull request by michaelryanpeter**
Version(s): 4.16+

Issue: [OCPBUGS-34836](https://issues.redhat.com/browse/OCPBUGS-34836)

Link to docs preview:
* https://80436--ocpdocs-pr.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-hybrid-helm.html#osdk-hh-project-layout_osdk-hybrid-helm

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by michaelryanpeter** [comment]
@emmajiafan PTAL

File: modules/osdk-hh-project-layout.adoc [hunk 1]
```diff
@@ -47,7 +47,7 @@ The Hybrid Helm Operator scaffolding is customized to be compatible with both He
 |`api/`
 |Contains the Go API definition.
 
-|`controllers/`
+|`internal/controllers/`
```
Comment (EricPonvelle) [inline modules/osdk-hh-project-layout.adoc:50]: Just noting that it renders fine in the preview:

image

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**